_includes/pbc/all/install-features/202307.0/install-the-service-points-customer-account-management-feature.md

```diff
@@ -0,0 +1,245 @@
+        ];
+    }
+}
+
+```
+
+**src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\ShopApplication;
+
+use SprykerShop\Yves\ServicePointWidget\Plugin\ShopApplication\ClickCollectServicePointAddressFormWidgetCacheKeyGeneratorStrategyPlugin;
+use SprykerShop\Yves\ShopApplication\ShopApplicationDependencyProvider as SprykerShopApplicationDependencyProvider;
+
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+    /**
+     * @return list<\SprykerShop\Yves\ShopApplicationExtension\Dependency\Plugin\WidgetCacheKeyGeneratorStrategyPluginInterface>
+     */
+    protected function getWidgetCacheKeyGeneratorStrategyPlugins(): array
+    {
+        return [
+            new ClickCollectServicePointAddressFormWidgetCacheKeyGeneratorStrategyPlugin(),
+        ];
+    }
+}
+```
+
+## 3) Set up widgets
+
+Register the following plugins to enable widgets:
+
+| PLUGIN                                    | SPECIFICATION                                                 | PREREQUISITES | NAMESPACE                                  |
+|-------------------------------------------|---------------------------------------------------------------|---------------|--------------------------------------------|
+| ClickCollectServicePointAddressFormWidget | Enables service point selection during checkout address step. |               | SprykerShop\Yves\ServicePointWidget\Widget |
+
+**src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\ShopApplication;
+
+use SprykerShop\Yves\ServicePointWidget\Widget\ClickCollectServicePointAddressFormWidget;
+use SprykerShop\Yves\ShopApplication\ShopApplicationDependencyProvider as SprykerShopApplicationDependencyProvider;
+
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+    /**
+     * @return list<string>
+     */
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            ClickCollectServicePointAddressFormWidget::class,
+        ];
+    }
+}
+```
+
+## 4) Set up FE part
+
+Adjust TWIG templates to display the service point selector.
+
+1. To the `page-layout-main` template ща `ShopUi` module, add the `main-overlay` molecule:
+
+```twig
+{% raw %}{% block globalComponents %}
+    ....
+    {% include molecule('main-overlay') only %}
+{% endblock %}{% endraw %}
+```
+
+2. Add `ClickCollectServicePointAddressFormWidget` to the `addres` view of the `CheckoutPage` module:
+
+```twig
+{% raw %}{% widget 'ClickCollectServicePointAddressFormWidget' args [data.checkoutAddressForm] only %}{% endwidget %}{% endraw %}
+```
+
+{% info_block infoBox "Info" %}
+
+In case using `ShipmentTypeAddressFormWidget` widget the `ClickCollectServicePointAddressFormWidget` will be added automatically, so no need to add it manually.
+
+{% endinfo_block %}
+
+3. Add `cross` icon to the `icon-spite` atom:
```

Comment:
Please add the module name and note that this is optional only if the project doesn't have this icon.
Please reorder the items to make changes in the `ShopUi` first.